php/test/Ice/enums/Client.php

```diff
@@ -10,10 +10,7 @@ function allTests($helper)
 {
     $ref = sprintf("test:%s", $helper->getTestEndpoint());
     $communicator = $helper->communicator();
-    $obj = $communicator->stringToProxy($ref);
-    test($obj != null);
-    $proxy = $obj->ice_checkedCast("::Test::TestIntf");
```

Comment:
I suspect we had all these ice_checkedCast and ice_uncheckedCast because it was the same with and without PHP namespace. Obviously, that's not the recommended API.